azerothcore/azerothcore-wotlk
# fix(DB/Creature): [ZG] Mad Servant (15111)

## Changes Proposed:
-  Populates creature_template_spell
-  Adds missing lines from Wowpedia found in broadcast_text
-  Updates creature SmartAI to include those lines

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11991

## SOURCE:


## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


1. `.learn 11726`
2. `.additem 6265 5`
3. `.go c 49060`
4. aggro a few, see if they say their 2 lines
5. cast Enslave Demon on a Mad Servant and check if they can cast Fireball and Flamestrike.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
